src/cascade/input_data/configuration/id_map.py

```diff
@@ -29,8 +29,8 @@
 14          15                    Cause-specific mortality rate mtspecific
 15          16                       Other cause mortality rate mtother
 16          17                               Case fatality rate
-17          18                                       Proportion
-18          19                                       Continuous
+17          18                                       Proportion Sincidence
```

Comment:
Really? This is for the single parameter models?

Reply:
That's where the data goes. It's for the single-parameter models, yes. This came out of a modeler meeting. What we then do with that data is another matter.